lib/at_monitor/Kconfig

```diff
@@ -14,8 +14,24 @@ config AT_MONITOR_HEAP_SIZE
 	range 64 4096
 	default 256
 
-config SYSTEM_WORKQUEUE_STACK_SIZE
-	default 1152 if (LTE_LINK_CONTROL && LOG)
+config AT_MONITOR_USE_DEDICATED_WORKQUEUE
```

Comment:
I'm hesitant to give application developers this option because of ~1,4kB of RAM out of which some can potentially be saved in system work queue stack size.

Reply:
This is enabled by default, but it's good to keep a it behind a flag for now. Some users might not need this (if they don't use `lte_lc` for example), and would rather save RAM instead.